readme.md

```diff
@@ -108,6 +108,15 @@ Shortcut for sending JSON. Use this instead of the `body` option. Accepts a plai
 
 Sets `options.method` to the method name and makes a request.
 
+#### baseUrl
```

Comment:
I know we went with `baseUrl` in Got, but hindsight, I think `baseURL` would have been better. Thoughts? // @szmarczak

Reply:
I'm comfortable with both solutions :)

Reply:
I kind of like capitalizing letters in acronyms, however there are some [good reasons not to](https://stackoverflow.com/questions/15526107/acronyms-in-camelcase).

> Difficulty in distinguishing multiple acronyms:
>    i.e. `USID` vs `usId` (or `parseDBMXML` ...

Reply:
> i.e. USID vs usId

Yup, but that doesn't apply in this case. Also, it would be `usID`, not `USID`.

Reply:
Right, the first example is bad, but the point is names with multiple all-caps acronyms next to each other make it unclear that there are actually two individual acronyms in there. Not a huge deal if you already happen to know the acronym. And of course, multi-acronym names are not super common and most people seeing this module will already know what a URL is, anyway. Still, I'd say `baseUrl` is a pretty strongly held conventional in JavaScript for the above reasons.

That said, let's be rebels. I will make it `baseURL`. :)

Reply:
Hmm, good points. I've changed my mind. If we add `baseURL`, we should really make `throwHttpErrors` be `throwHTTPErrors`, but then we have the problem of not having clear separation. Probably easier to just always stay with camelCase no matter what. So let it be `baseUrl`.

Reply:
I just noticed that `ky.HTTPError` is using the all-caps style. Want to rename it? (Separate PR, of course.)